platforms/core-configuration/kotlin-dsl/src/main/kotlin/org/gradle/kotlin/dsl/PluginDependenciesSpecExtensions.kt

```diff
@@ -25,11 +25,27 @@ import org.gradle.plugin.use.PluginDependencySpec
  *
  * Visit the [Build Scan Plugin User Manual](https://docs.gradle.com/build-scan-plugin/) for additional information.
  *
- * By default, the applied plugin version will be the same as the one used by the `--scan` command line option.
+ * By default, the applied plugin version will be "3.16.2", which is the last version of the Gradle Enterprise plugin.
+ * To apply later versions, use `develocity` plugin instead.
  *
  * You can also use e.g. `` `gradle-enterprise` version "3.0" `` to request a different version.
  *
  * @since 6.0
  */
 val PluginDependenciesSpec.`gradle-enterprise`: PluginDependencySpec
+    get() = this.id(AutoAppliedGradleEnterprisePlugin.GRADLE_ENTERPRISE_PLUGIN_ID.id).version(AutoAppliedGradleEnterprisePlugin.LAST_GRADLE_ENTERPRISE_PLUGIN_VERSION)
+
+
+/**
+ * The `develocity` plugin.
+ *
+ * Visit the [Build Scan Plugin User Manual](https://docs.gradle.com/enterprise/gradle-plugin/) for additional information.
```

Comment:
Let's use the new name/URL here wdyt? `[Develocity Gradle Plugin User Manual](https://docs.gradle.com/enterprise/gradle-plugin/)`

Reply:
Good catch, fixed in https://github.com/gradle/gradle/pull/28673/commits/173282a6df614b11c141ed238c06a1401bd40a3c